q2_annotate/busco/utils.py

```diff
@@ -180,9 +222,12 @@ def _parse_df_columns(df: pd.DataFrame) -> pd.DataFrame:
         df (pd.DataFrame): Formatted DataFrame
     """
     cols = MARKER_COLS.copy()
+
     if not ("completeness" in df.columns and "contamination" in df.columns):
```

Comment:
Can you change this part and the contents of MARKER_COLS so that you _add_ here instead of remove? IT would then be consistent with the second branch of this statement.